api/all/src/main/java/io/opentelemetry/api/logs/EventBuilder.java

```diff
@@ -0,0 +1,53 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package io.opentelemetry.api.logs;
+
+import io.opentelemetry.api.common.Attributes;
+import io.opentelemetry.api.common.Value;
+import io.opentelemetry.context.Context;
+import java.time.Instant;
+import java.util.concurrent.TimeUnit;
+
+/** The EventBuilder is used to {@link #emit()} events. */
+public interface EventBuilder {
```

Comment:
I'm curious what would be the difference between `EventBuilder` and `LogRecordBuilder` - they seem to be identical.

The only difference seem to be in the implementation type or even in the impl constructor argument.

The minimalistic version would be to emit event with `logger.eventBuilder(eventName)` and log record with `logger.logRecordBuilder()` but use the same implementation and builder types.

I wonder if we really need all this extra stuff.